core/test/Test/Pos/Core/Gen.hs

```diff
@@ -139,6 +140,11 @@ module Test.Pos.Core.Gen
         -- Pos.Merkle Generators
         , genMerkleRoot
         , genMerkleTree
+
+        -- Helpers
+        , genMillisecond
```

Comment:
I believe there's a `genMillisecond` defined and exposed from `util-test`. Also, it'd probably be a good idea for us to move some of these helper gens to `util-test` at some point, but it's not required right now. 

We can just leave this as is.

Reply:
Cheers! I clearly didn't see that generator in `util`. It's currently by itself, maybe a follow up PR to just move the rest of the "neutral" gens there? I'm indifferent about this.